<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>479</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x0:1, y1:0, x2:1, y2:0, stop:0 rgb(60, 227, 227), stop:1 rgba(0, 0, 128, 255));
border-radius:100px;</string>
  </property>
  <widget class="QLabel" name="ScoreText">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>50</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trafic Mazar</family>
     <pointsize>18</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
color:rgb(0, 0, 0);
background-color:rgb(255, 213, 88);
border-radius:10px</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Score : 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="TimeText">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trafic Mazar</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
color: rgb(0, 0, 0);
background-color:rgb(255, 213, 88);
border-radius:10px;
font:22pt &quot;Trafic Mazar&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;Time passed : 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background-color: rgba(255, 255, 255, 0);
border: 0;
color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(170, 0, 0);
color: rgb(255,255,255);
border-radius:10px
}
</string>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PauseButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>400</y>
     <width>110</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>110</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Black</family>
     <pointsize>24</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton {
background-color: rgba(255, 255, 255, 0);
border: 0;
color: rgb(255, 255, 255);
}
QPushButton:hover{
background-color: rgb(0, 199, 0);
color: rgb(255,255,255);
border-radius:10px;
}
</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="FigureWindow">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>290</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
border-left: 0px;
color: rgba(0,0,0,0);
border-radius:10px;
background-color: rgb(255, 255, 255)



</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="cornerButtonEnabled">
    <bool>false</bool>
   </property>
   <property name="rowCount">
    <number>5</number>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="textFigure">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>140</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>100</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>TF2 Secondary</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(0, 0, 0);
Border: 1px solid grey;
border-right: 0px;
border-radius:10px;
background-color:rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>Next figure : </string>
   </property>
  </widget>
  <widget class="QLabel" name="LevelText">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>170</y>
     <width>240</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>240</width>
     <height>50</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trafic Mazar</family>
     <pointsize>22</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
background-color:rgb(255, 213, 88);
color:rgb(0, 0, 0);
border-radius:10px;
font:22pt &quot;Trafic Mazar&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:700;&quot;&gt;Level : 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="LinesText">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>230</y>
     <width>241</width>
     <height>50</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1</width>
     <height>1</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>999</width>
     <height>999</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Trafic Mazar</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>IBeamCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">Border: 1px solid grey;
padding: 10px;
color:rgb(0, 0, 0);
background-color:rgb(255, 213, 88);
border-radius:10px;
font:18pt &quot;Trafic Mazar&quot;;

</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;Lines : 0&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="PlayTable">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>300</width>
     <height>400</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>300</width>
     <height>400</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>2</pointsize>
    </font>
   </property>
   <property name="cursor" stdset="0">
    <cursorShape>CrossCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgba(0,0,0,0);
border: 1px solid grey;
text-indent: 50px;
letter-spacing: -10px;
line-height: 0.8;
white-space: nowrap;
padding: 0;
margin: 0;
border-radius:none;
border:none;
background-color:white
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContents</enum>
   </property>
   <property name="autoScroll">
    <bool>false</bool>
   </property>
   <property name="autoScrollMargin">
    <number>0</number>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
   <property name="showDropIndicator" stdset="0">
    <bool>false</bool>
   </property>
   <property name="dragDropOverwriteMode">
    <bool>false</bool>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::NoSelection</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="gridStyle">
    <enum>Qt::DashLine</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="rowCount">
    <number>20</number>
   </property>
   <property name="columnCount">
    <number>15</number>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="horizontalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="verticalHeaderMinimumSectionSize">
    <number>0</number>
   </attribute>
   <attribute name="verticalHeaderDefaultSectionSize">
    <number>20</number>
   </attribute>
   <attribute name="verticalHeaderHighlightSections">
    <bool>false</bool>
   </attribute>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <row/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <column/>
   <item row="0" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(82, 108, 255)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(245, 71, 71);
}
QPushButton:hover {
	background-color: rgb(245, 71, 71, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_minimize">
    <property name="geometry">
     <rect>
      <x>558</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(254, 188, 69);
}
QPushButton:hover {
	background-color: rgb(254, 188, 69, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_maximize">
    <property name="geometry">
     <rect>
      <x>536</x>
      <y>10</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
	border: none;
	border-radius: 8px;
	background-color: rgb(50, 196, 50);
}
QPushButton:hover {
	background-color: rgb(50, 196, 50, 150);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
